### PR TITLE
Saving changes to donor homepage config

### PR DESCRIPTION
**Pull request by bonnie-chin**
**Work Done:**
- Modified initialization of React context such that it updates when query is complete
- - Added spinner while query loading
- Used mutation to save changes to the config on ui
- Added success alert upon mutation

Miscellaneous:
- Adjusted word wrapping on each testimonial card 
- Fixed seeder to delete previous donorHomepage items made

Figma:
image

Implementation (will fix colours shortly): 
image

### CODE REVIEW

File: client/src/data/types/donorHomepageConfig.ts [hunk 1]
```diff
@@ -30,12 +30,6 @@ export interface DonorHomepageConfig {
         testimonials: Array<Testimonial>;
     };
     statistics: Array<Statistic>;
-    banner: {
-        header: string;
-        description: string;
-        imagePaths: Array<string>;
-        interval: number;
-    };
```
Comment (Puepis) [inline client/src/data/types/donorHomepageConfig.ts:38]: how come banner isn't needed?

File: client/src/pages/AdminEditTestimonialsPage.tsx [hunk 2]
```diff
@@ -15,47 +16,151 @@ export type EditTestimonialsFormState = {
...
+
+    useEffect(() => {
+        if (!showSuccessAlert) return;
+        setTimeout(() => {
+            setShowSuccessAlert(false);
+        }, 5000);
+    }, [showSuccessAlert])
 
     const handleSave = () => {
         console.log("save");
```
Comment (Puepis) [inline client/src/pages/AdminEditTestimonialsPage.tsx:109]: nit: stray console log

**Review by Puepis — APPROVED** [review]
tested it and it looks great! awesome work :))

File: client/src/data/types/donorHomepageConfig.ts (hunk 1 above)
Comment (bonnie-chin) [reply client/src/data/types/donorHomepageConfig.ts:38]: oh just cause we don't edit it on this page, or ever

**Pull request merged**